applications/ParticleMechanicsApplication/ParticleMechanicsApplication.json

```diff
@@ -2,7 +2,7 @@
 	"wheel_name": "KratosParticleMechanicsApplication",
 	"included_modules": ["ParticleMechanicsApplication"],
 	"included_binaries": ["KratosParticleMechanicsApplication.*", "KratosParticleMechanicsCore.*", "libKratosParticleMechanicsCore.*"],
-	"dependencies": ["KratosMultiphysics==${KRATOS_VERSION}"],
+	"dependencies": ["KratosMultiphysics==${KRATOS_VERSION}", "KratosStructuralMechanicsApplication==${KRATOS_VERSION}", "KratosLinearSolversApplication==${KRATOS_VERSION}"],
```

Comment:
I thought mpm was independent of structural?
At least we had a long discussion abt this
@tteschemacher ?

Reply:
That's actually interesting. I did think the same. Not sure who added this...

Reply:
It could be though that there are some python dependencies that I'm not aware of.

Reply:
I agree there should be no dependency  to StructuralMechanicsApp. @roigcarlo why is this change needed?

Reply:
I didn't find any mentioning of the StructuralApp in the ParticleApp either

Reply:
I followed the dependencies of @jginternational PR https://github.com/KratosMultiphysics/GiDInterface/pull/859

Reply:
I think we reached a confussion.

That dependency is just a GUI related one (conditions)

The binary dependency is the python_packages field @roigcarlo

Reply:
aaaaa ok

Reply:
```suggestion
	"dependencies": ["KratosMultiphysics==${KRATOS_VERSION}", "KratosLinearSolversApplication==${KRATOS_VERSION}"],
```